rust/parquet/src/arrow/arrow_writer.rs

```diff
@@ -1049,9 +1049,7 @@ mod tests {
     }
 
     #[test]
-    #[should_panic(
-        expected = "Reading parquet list array into arrow is not supported yet!"
-    )]
+    #[ignore = "Roundtrip failing, reason likely reader, not yet investigated"]
```

Comment:
@carols10cents when you can, may you please help me out here on these 2 tests that I've ignored?
One needs a `LargeList` writer implementation, which I think you'd do better than me as you've done roundtrip for other Arrow types.

I've unignored `struct_single_column` as it's now passing, might be the null comparison commit that fixed it.

Thanks :smiley:

Reply:
Looking at this now!

Reply:
@nevi-me I [opened a PR on your repo](https://github.com/nevi-me/arrow/pull/6) with some further work -- feel free to cherry pick or whatever, but the tests aren't quite passing yet over there, just failing for a different reason ;)